_i18n/de/resources/moneropedia/transaction.md

```diff
@@ -1,22 +1,23 @@
 ---
-terms: ["transaction", "transactions"]
-summary: "a cryptographically signed container that details the transfer of Monero to a recipient (or recipients)"
+terms: ["transaction", "transactions", "Transaktion", "Transaktionen"]
+summary: "Ein kryptografisch signierter Behälter, der einen Monero-Transfer einem (oder mehreren) Empfänger(n) zuweist"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A cryptographically signed container that details the transfer of Monero to a recipient (or recipients).
+Ein kryptografisch signierter Behälter, der einen Monero-Transfer einem (oder mehreren) Empfänger(n) zuweist.
 
-The parameters of a transaction contain one or more recipient addresses with corresponding amounts of funds and a @ring-size parameter that specifies the number outputs bound to the transaction. The more outputs that are used, a higher degree of obfuscation is possible, but that comes with a cost. Since a transaction gets larger with more outputs, the transaction fee will be higher.
+Die Parameter einer Transaktion enthalten eine oder mehrere Empfängeradressen mit entsprechenden Geldbeträgen und einen Parameter bezüglich der @Ringgröße, welcher die Anzahl der an die Transaktion gebundenen Outputs festlegt. Je mehr Outputs verwendet werden, desto höher kann der Grad an Verschleierung sein. Dies hat jedoch seinen Preis: Da eine Transaktion durch mehr Outputs größer wird, werden die Transaktionsgebühren höher ausfallen.
 
-It is possible to form a transaction offline, which offers additional privacy benefits.
+Die Offline-Erstellung einer Transaktion ist möglich und bietet zusätzliche, die Privatsphäre begünstigende Vorteile.
 
-A transaction can be uniquely identified with the use of an optional Transaction ID, which is usually represented by a 32-byte string (64 hexadecimal characters).
+Eine Transaktion kann durch den Gebrauch einer optionalen Transaktions-ID eindeutig gekennzeichnet werden. Diese wird in der Regel durch einen 32-Byte-Zeichenkette (oder 64 Hexadezimalzeichen) verkörpert.
 
-### In-depth Information
-Every transaction involves two keys: a public @spend-key, and a public @view-key. The destination for an output in a transaction is actually a one-time public key computed from these two keys.
+### Ausführliche Informationen
 
-When a wallet is scanning for incoming transactions, every transaction is scanned to see if it is for "you". This only requires your private view key and your public spend key, and this check is immutable and cannot be faked. You cannot receive transactions and identify them without a corresponding private view key.
+Jede Transaktion beinhaltet zwei Schlüssel: Einen öffentlichen @Spend-Key und einen öffentlichen @View-Key. Das Ziel eines Outputs einer Transaktion ist genau genommen ein öffentlicher Einmal-Schlüssel, der aus diesen zwei Schlüsseln berechnet wurde.
```

Comment:
.. not sure if "beinhaltet / enthält" ("contains") is the correct translation of "involves" here?

Reply:
We could simply use "involviert" here?

Reply:
Yes that'd be better!